seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2647,6 +2647,48 @@
       }
     },
     {
+      "id": "cxk61uhxor7m4a1swj1ha9eo",
```

Comment:
Thanks @QuincyLarson for your quick comment. I was also thinking about the difficulty of this challenge and I think in the context of previous challenges (mainly this very previous challenge) it gets easier.

I will change the id in the next commit.